# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../etcnet/0001-removed-unintentionally-left-debug-line.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From 221cbec3b0f5c83e1bc64209fd2068dad4fd3283 Mon Sep 17 00:00:00 2001
From: Sergey Bolshakov <sbolshakov@altlinux.org>
Date: Sun, 17 Jan 2010 17:32:00 +0300
Subject: [PATCH 1/4] removed unintentionally left debug line

---
 etc/net/scripts/ifdown-removable |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/etc/net/scripts/ifdown-removable b/etc/net/scripts/ifdown-removable
index 469d5f4..cfe38a8 100755
--- a/etc/net/scripts/ifdown-removable
+++ b/etc/net/scripts/ifdown-removable
@@ -2,8 +2,6 @@
 
 # This script is a hook for hotplug or pcmcia-cs.
 
-/sbin/ip ad > /tmp/ipad
-
 usage()
 {
 	echo "Usage: $0 <interface>" >&2
-- 
1.6.6.2

